code/modules/trade/datums/trade_stations_presets/3-rare/monstertrapper.dm

```diff
@@ -47,7 +47,7 @@
 			/obj/item/reagent_containers/glass/bottle/trade/fuhrerole = good_data("fuhrerole bottle", list(1, 1), 900),
 			/obj/item/reagent_containers/glass/bottle/trade/kaiseraurum = good_data("kaiseraurum bottle", list(1, 1), 1000)
 		),
-		"Just Spiders" = list(
+		"Live Spiders" = list(
```

Comment:
```suggestion
		"Just Spiders" = list(
```
Its to funny to change, its the way it is

Reply:
That’s not funny in any way. I’m honestly surprised you want to keep the „just spiders“ but gutted every flavor text from food items that had some degree of funny. Where do you draw the line of funny and flavor to keep or what to throw out?